src/Quasar/QuasarF.purs

```diff
@@ -161,17 +170,25 @@ invokeFile
   → FilePath
   → Vars
   → Maybe Pagination
-  → QuasarFE JArray
+  → QuasarFE (ExpiredContent JArray)
 invokeFile mode path vars pagination =
   InvokeFile mode path vars pagination id
 
 invokeFileEJson
   ∷ FilePath
   → Vars
   → Maybe Pagination
-  → QuasarFE (Array EJson)
+  → QuasarFE JArray
 invokeFileEJson path vars pagination =
-  invokeFile Precise path vars pagination <#> resultsAsEJson
+  map _.content <$> invokeFileEJsonDetail path vars pagination
```

Comment:
just spotted I don't have `<#> resultsAsEJson` and not actually sure I need to lift out `_.content`

Reply:
`resultsAsEJson` part should be handled in `invokeFileEJsonDetail`